state.go

```diff
@@ -737,7 +742,7 @@ func (s *State) Handle(pt *MLSPlaintext) (*State, error) {
 
 	// Advance the key schedule
 	next.Epoch += 1
-	next.updateEpochSecrets(next.TreePriv.UpdateSecret)
+	next.updateEpochSecrets(commitSecret)
 
```

Comment:
so that i understand it right, if there is no commit path provided, we updateEpochSecrets with all-zeroes commitSecret ..isn't it ?

Reply:
Correct.